.github/workflows/build.yml

```diff
@@ -6,6 +6,9 @@ jobs:
     strategy:
       matrix:
         cc: [gcc, clang]
+        release_type:
```

Comment:
Maybe rename this? Otherwise looks fine.

Reply:
Isn't it already named `release_type` ? IMO self explanatory.

Reply:
It makes more sense to be called `build_type`.

Reply:
Yes, `build_type` for consistency please.

Reply:
done